src/sharepoint/rest/actionables.ts

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+import { Queryable } from "./queryable";
+import * as Mixins from "./mixins";
+import * as Util from "../../utils/util";
+
+/**
+ * A Queryable which only exposes the get method
+ * 
+ */
+export class Gettable<T> extends Queryable implements Mixins.Gettable {
+
+    /**
+     * Creates a new instance of the Gettable class
+     * 
+     * @param baseUrl The url or Queryable which forms the parent of this actionable
+     */
+    constructor(baseUrl: string | Queryable) {
+        super(baseUrl);
+    }
+
+    /**
+     * Execute the get request
```

Comment:
For what, the constructors? We could decide as a team to not comment constructors or other specific methods. I commented everything as I was going through.